users/views.py

```diff
@@ -17,7 +17,17 @@
 from rest_framework.decorators import api_view, renderer_classes
 from rest_framework.renderers import JSONRenderer, TemplateHTMLRenderer
 from bookmark.models import Bookmark
-#@csrf_exempt
+from users.serializers import UserViewSerializer
+# @csrf_exempt
+
+class UserView(APIView):
+
+    def get(self, request):
+        user = IsLoggedIn(request)
```

Comment:
Use a try except block to gracefully return exceptions

Reply:
Done